### PR TITLE
Document the implied 'Default' property group when no group is specified

### PR DESCRIPTION
**Pull request by akoebbe**
Pretty sure the example code should be referencing `details` and not `Default`.

### CODE REVIEW

**Comment by goetas** [comment]
Pay attention to https://github.com/schmittjoh/serializer/blob/master/src/JMS/Serializer/Exclusion/GroupsExclusionStrategy.php#L34

The documentation says that there is a `Default` group  :)

**Comment by akoebbe** [comment]
@goetas Ah ha! Thanks for pointing that out. It wasn't apparent that the `Default` group existed. Where is that in the docs?

**Comment by goetas** [comment]
i guess the document you tried to change, had the intent do document it :)

but an explicit section does not exists probably... interested in adding it?

**Comment by akoebbe** [comment]
Yup. I'll back out the last commit and add an explicit statement about it.

**Comment by akoebbe** [comment]
@goetas let me know if that is clear enough.

File: doc/cookbook/exclusion_strategies.rst [hunk 1]
```diff
@@ -86,7 +86,9 @@ Another default exclusion strategy is to create different views of your objects.
 Let's say you would like to serialize your object in a different view depending
 whether it is displayed in a list view or in a details view.
 
-You can achieve that by using the ``@Groups`` annotation on your properties.
+You can achieve that by using the ``@Groups`` annotation on your properties. Any
+property without an explicit ``@Groups`` annotation will be included in a
+``Default`` group, which can be used when specifying groups in the serializer.
```
Comment (goetas) [inline doc/cookbook/exclusion_strategies.rst]: "...groups in the serialization context." ?

**Comment by goetas** [comment]
thanks

**Pull request merged**